backend/src/packages/chaiNNer_standard/image_dimension/utility/get_bounding_box.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import Tuple
+
+import numpy as np
+
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import NumberOutput
+from nodes.utils.utils import get_h_w_c
+
+from .. import utility_group
+
+
+@utility_group.register(
+    schema_id="chainner:image:get_bbox",
+    name="Get Bounding Box",
+    description="Gets a bounding box (X, Y, Height, and Width) of the white area of a mask.",
+    icon="BsRulers",
+    inputs=[
+        ImageInput(channels=1),
+        SliderInput(
+            "Threshold",
+            precision=1,
+            minimum=1,
+            controls_step=1,
+            slider_step=1,
+            default=1,
+        ),
+    ],
+    outputs=[
+        NumberOutput("X"),
+        NumberOutput("Y"),
+        NumberOutput("Width"),
+        NumberOutput("Height"),
+    ],
+)
+def get_dimensions_node(
```

Comment:
It's still the old name.
```suggestion
def get_bounding_box_node(
```

Reply:
Ah yeah, sorry I forgot xd